phabricatoremails/query_position_store.py

```diff
@@ -3,21 +3,30 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from dataclasses import dataclass
+from typing import Protocol
 
 from phabricatoremails.models import QueryPosition
 from sqlalchemy.orm import Session
 
 
+class QueryPositionStore(Protocol):
+    def get_position(self) -> QueryPosition:
+        ...
```

Comment:
nit: since you use `pass` elsewhere in the code, it would be cleaner to be consistent and use it here, instead of `...`. There's also an opportunity here to add some docstrings, both to this and `set_initial_position`.

Reply:
Huh, in [the PEP](https://www.python.org/dev/peps/pep-0544/#id18) they always use `...`, but TIL they're equivalent. `pass` it is!